app/javascript/packs/leafletMap.js

```diff
@@ -64,14 +64,42 @@ export async function setupMap() {
   map.on("click", onClick);
 }
 
+function parsePosition(position){
```

Comment:
This doesn't really "parse" anything, it constructs the query parameter string for the given position according to the interface our backend defines